doc/qbk/6.2.MutableString.qbk

```diff
@@ -0,0 +1,70 @@
+[/
+    Copyright (c) 2021 Vinnie Falco (vinnie.falco@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/CPPAlliance/url
+]
+
+[section:mutablestring MutableString]
+
+A ['MutableString] is a container that has the `assign` and
+`append` member functions that accept iterators to `char`.
+
+The library __decoded_view__ class provides a operations
```

Comment:
grammar

Reply:
What?

Reply:
The whole sentence is a train wreck lol

Reply:
Oh... You thought you were talking about the `grammar` namespace in your first comment! :laughing: